agents/node/vcxagent-core/src/agent.js

```diff
@@ -32,12 +34,16 @@ async function createVcxAgent ({ agentName, genesisPath, agencyUrl, seed, usePos
   async function agentInitVcx () {
```

Comment:
Since. this is now used by all integrations tests (I think all of them rely on vcxagent-core), could you also add a test which would just verify that the old workflow also still works? (initVcxCore+openVcxPool+openVcxWallet)

Then in some next PR, let's also add java+ios wrappers for the functions added in this PR.